epp-client/src/common.rs

```diff
@@ -303,7 +303,26 @@ pub struct Address {
     pub postal_code: StringValue,
     /// The &lt;cc&gt; tag under &lt;addr&gt;
     #[serde(rename = "contact:cc", alias = "cc")]
-    pub country_code: StringValue,
+    pub country: Country,
```

Comment:
It might be nicer to use `celes::Country` here directly, but that seems to serialize into an attribute instead of an element, and the `$unflatten` trick did not seem to help with that.